src/Microsoft.Data.SqlClient/netcore/ref/Microsoft.Data.SqlClient.cs

```diff
@@ -437,70 +240,105 @@ public sealed partial class SqlCommand : System.Data.Common.DbCommand, System.IC
+        [System.ComponentModel.RefreshPropertiesAttribute(System.ComponentModel.RefreshProperties.All)]
+        [System.ComponentModel.SettingsBindableAttribute(true)]
         public override string ConnectionString { get { throw null; } set { } }
+        [System.ComponentModel.DesignerSerializationVisibilityAttribute(0)]
         public override int ConnectionTimeout { get { throw null; } }
+        [System.ComponentModel.BrowsableAttribute(false)]
+        [System.ComponentModel.DesignerSerializationVisibilityAttribute(0)]
+        public Microsoft.Data.SqlClient.SqlCredential Credential { get { throw null; } set { } }
+        [System.ComponentModel.DesignerSerializationVisibilityAttribute(0)]
         public override string Database { get { throw null; } }
+        [System.ComponentModel.BrowsableAttribute(true)]
+        [System.ComponentModel.DesignerSerializationVisibilityAttribute(0)]
         public override string DataSource { get { throw null; } }
         public bool FireInfoMessageEventOnUserErrors { get { throw null; } set { } }
+        [System.ComponentModel.DesignerSerializationVisibilityAttribute(0)]
         public int PacketSize { get { throw null; } }
+        [System.ComponentModel.BrowsableAttribute(false)]
+        [System.ComponentModel.DesignerSerializationVisibilityAttribute(0)]
         public override string ServerVersion { get { throw null; } }
+        [System.ComponentModel.BrowsableAttribute(false)]
+        [System.ComponentModel.DesignerSerializationVisibilityAttribute(0)]
         public override System.Data.ConnectionState State { get { throw null; } }
+        [System.ComponentModel.DefaultValueAttribute(false)]
         public bool StatisticsEnabled { get { throw null; } set { } }
+        [System.ComponentModel.DesignerSerializationVisibilityAttribute(0)]
         public string WorkstationId { get { throw null; } }
-        public Microsoft.Data.SqlClient.SqlCredential Credential { get { throw null; } set { } }
         public event Microsoft.Data.SqlClient.SqlInfoMessageEventHandler InfoMessage { add { } remove { } }
         protected override System.Data.Common.DbTransaction BeginDbTransaction(System.Data.IsolationLevel isolationLevel) { throw null; }
         public new Microsoft.Data.SqlClient.SqlTransaction BeginTransaction() { throw null; }
         public new Microsoft.Data.SqlClient.SqlTransaction BeginTransaction(System.Data.IsolationLevel iso) { throw null; }
         public Microsoft.Data.SqlClient.SqlTransaction BeginTransaction(System.Data.IsolationLevel iso, string transactionName) { throw null; }
         public Microsoft.Data.SqlClient.SqlTransaction BeginTransaction(string transactionName) { throw null; }
         public override void ChangeDatabase(string database) { }
+        public static void ChangePassword(string connectionString, Microsoft.Data.SqlClient.SqlCredential credential, System.Security.SecureString newSecurePassword) { }
```

Comment:
How come `throw null;` was removed?

Reply:
Not sure. There were a bunch of other methods near it that didn't throw either so I've added it to a few. I'm going to see if can track down the exact rules for this since my knowledge is based on seeing ref edits and review feedback on my own.

Reply:
I am a bit confused, too, why some include it and some don't. So I'm curious what you find out. I don't feel like it's a blocker, though, given my understand of ref.